extern/googlebenchmark/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Algebra plugins library, part of the ACTS project (R&D line)
+#
+# (c) 2021 CERN for the benefit of the ACTS project
```

Comment:
2022

Reply:
Ah, good catch.

Reply:
Here, I actually copied the beginning portion (that is, the copyright stuff) from the [`extern/googletest`](https://github.com/acts-project/algebra-plugins/blob/d94ef468209a0c6eadb67130bf50ca37a19a40e7/extern/googletest/CMakeLists.txt#L3) CMakeLists.txt. It has 2021 there too.

Should I change it in this PR, or make a new PR for it? It looks to me like there are quite a few places which have the copyright line still on 2021.

Reply:
You can change it in this PR. In general, when you add a new file, the copyright should be set to the year it is deployed. If you modify old code in existing files, the copyright will be extended to the year it was modified e.g. ```2021-2022```.